integr8ly/integreatly-operator
# MGDAPI-4548 fix the URLs and marin3r pod status check

# Issue link
https://issues.redhat.com/browse/MGDAPI-4548
https://issues.redhat.com/browse/MGDAPI-4592

# What
- update to move all sops to gitlab instead of github repo
- small update to the logic around confirming marin3r pods state

# Verification steps
- Install RHOAM 1.26.0 via OLM
- Upgrade to RHOAM 1.27.0 using this branch as the source of the container image for 1.27.0
- Navigate to Observe > alerting in OSD and confirm that RHOAM alerts point to gitlab instead of github
- Navigate to Prometheus (you can get the route from observability namepspace) and go through each alert and confirm that they all have been moved to GitLab and no more github alerts are present.
- Logic/code check on rate limit pods
